src/main/java/com/example/securitynew/exception/RegistrationException.java

```diff
@@ -4,4 +4,5 @@ public class RegistrationException extends Exception {
     public RegistrationException(String message) {
         super(message);
     }
+
```

Comment:
make it unchecked, cause you dont use it as checked.

Reply:
not fixed